packages/load-cfg/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@types/find-up": "^2.1.1",
-    "@types/node": "^10.3.2"
+    "@types/node": "^10.3.2",
+    "libundler": "^1.7.1"
```

Comment:
I don't need to add `libundler` as dependency, we're using lerna. With lerna you can have `devDependencies` just in the root package! But you need to run `lerna bootstrap`

Reply:
got it, thanks!